addons/l10n_ch/migrations/11.0.10.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,23 @@
+---Fields in module 'l10n_ch'---
+l10n_ch      / account.invoice          / l10n_ch_isr_sent (boolean)    : NEW
+l10n_ch      / res.bank                 / l10n_ch_postal_chf (char)     : NEW
+l10n_ch      / res.bank                 / l10n_ch_postal_eur (char)     : NEW
+l10n_ch      / res.company              / l10n_ch_isr_print_bank_location (boolean): NEW
+# Nothing to do
+
+---XML records in module 'l10n_ch'---
+NEW account.tax.group: l10n_ch.tax_group_tva_0 (noupdate)
+NEW account.tax.group: l10n_ch.tax_group_tva_25 (noupdate)
+NEW account.tax.group: l10n_ch.tax_group_tva_38 (noupdate)
+NEW account.tax.group: l10n_ch.tax_group_tva_8 (noupdate)
```

Comment:
You have to try to remove these records.

Reply:
@pedrobaeza 
done!